.github/workflows/lint.yml

```diff
@@ -20,8 +20,10 @@ jobs:
           node-version: 10.x
       - name: Set up ESLint
         run: npm install --global eslint
+      - name: Set up ESLint html plugin
+        run: npm install eslint-plugin-html@latest --save-dev
       - name: Run ESLint checks
-        run: eslint .
+        run: eslint --ext .njk,.js data/ lib/ templates/
```

Comment:
We were using `.eslintignore` to filter out the folders to be linted, so we wouldn't need to use the paths here. I'll take a look into it.